README.md

```diff
@@ -1,6 +1,36 @@
 # gnag <a href="https://travis-ci.org/btkelly/gnag"><img src="https://travis-ci.org/btkelly/gnag.svg" /></a>&nbsp;<a href='https://bintray.com/btkelly/maven/gnag-gradle-plugin/_latestVersion'><img src='https://api.bintray.com/packages/btkelly/maven/gnag-gradle-plugin/images/download.svg'></a>
 A Gradle plugin that helps facilitate Github PR checking and automatic commenting of violations.
 
+# Usage
+
+The simplest way to use this plugin is in conjunction with the <a href="https://github.com/noveogroup/android-check">android-check</a> plugin. This allows you to integrate checkstyle, findbugs, and PMD with minimal effort. It is important to leave the report output paths as default and disable the `abortOnError` flag to allow gnag to finish collecting all reports.
+
+Add the following to the build script of your project
+
+```
+buildscript {
+    repositories {
+        maven {
+            jcenter()
+        }
+    }
+    dependencies {
+        classpath 'com.btkelly:gnag:{current version}'
+    }
+}
+
+apply plugin: 'gnag-plugin'
+
+gnag {
+    gitHubRepoName 'user/repo'
+    gitHubAuthToken "12312n3j12n3jk1"
+    gitHubIssueNumber "11"
+    failBuildOnError true
+}
+```
+
+This is the simplest way to add automatic PR checking and commenting to your project. The options above can be overridden by passing command line parameters with the same name to your build. This is helpful when using in conjunction with a CI system to allow automated builds.
```

Comment:
"The options above" - might be good to specifically call out that this refers to the options set in the gnag closure (iirc?)